src/github.com/matrix-org/dendrite/federationapi/writers/send.go

```diff
@@ -159,7 +159,7 @@ func (t *txnReq) processEvent(e gomatrixserverlib.Event) error {
 	// TODO: Check that the event is allowed by its auth_events.
 
 	// pass the event to the roomserver
-	if err := t.producer.SendEvents([]gomatrixserverlib.Event{e}); err != nil {
+	if err := t.producer.SendEvents([]gomatrixserverlib.Event{e}, ""); err != nil {
```

Comment:
Either add a one-liner here or use a self-documenting constant name please. The constant name could be "cute", maybe.